### ISSUE TITLE
start(opts, cb) is requiring drivers not in options

### ISSUE THREAD

**Issue by ADSKLowenthal** [vvo/selenium-standalone#128]
I'm using this code to start selenium server from Gulp
```js
  selenium.start({
    drivers: {
      chrome: {
        version: '2.15',
        arch: process.arch,
        baseUrl: 'http://chromedriver.storage.googleapis.com'
      }
    }
  }, function(err, childProc){
    console.log(err);
    child = childProc;
    done();
  })
```

The error I'm receiving says it's missing IE Driver, which I don't want to use and have no included in my start options.

```
[Error: Missing D:\dev\projects\gamedev-website\node_modules\selenium-standalone\.selenium\iedriver\2.47.0-x64-IEDriverServer.exe]
```

Install Script
```js
//Install selenium server for use with Protractor End to End testing
selenium.install({
  // check for more recent versions of selenium here:
  // http://selenium-release.storage.googleapis.com/index.html
  version: '2.47.1',
  baseURL: 'http://selenium-release.storage.googleapis.com',
  drivers: {
    chrome: {
      // check for more recent versions of chrome driver here:
      // http://chromedriver.storage.googleapis.com/index.html
      version: '2.15',
      arch: process.arch,
      baseURL: 'http://chromedriver.storage.googleapis.com'
    },
    ie: {
      // check for more recent versions of internet explorer driver here:
      // http://selenium-release.storage.googleapis.com/index.html
      version: '2.45',
      arch: process.arch,
      baseURL: 'http://selenium-release.storage.googleapis.com'
    }
  },
  logger: function(message) {
    console.log(message);
  },
  progressCb: function(totalLength, progressLength, chunkLength) {

  }
}, function(err){});
```

**Comment by vvo** [comment]
I do not have much time to look into this, can you make sure to use the latest version and debug it? code is straightforward. thanks

**Comment by vvo** [comment]
Indeed we should not merge passed options with defaults, accepting a PR to fix this

**Comment by vvo** [comment]
solved in 5.0.0
